test/Turn-test.js

```diff
@@ -0,0 +1,77 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Card = require('../src/Card');
+const Turn = require('../src/Turn');
+
+describe('Turn', function() {
+
+  it('should be a function', function() {
+    const turn = new Turn();
+
+    expect(Turn).to.be.a('function');
+  });
+
+  it('should be an instance of Turn', function() {
+    const turn = new Turn();
+
+    expect(turn).to.be.an.instanceOf(Turn);
+  });
+
+  it('should take a guess as the first argument', function() {
+    const card = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object');
+    const turn = new Turn('function');
+
+    expect(turn.guess).to.equal('function');
+  });
+
+  it('guess argument should be a string', function() {
+    const card = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object');
+    const turn = new Turn('function');
+
+    expect(turn.guess).to.be.a('string');
```

Comment:
I originally had an additional test to return an error when a non-string argument is passed in for the guess parameter.  I'm wondering if I should add it back in.  The main reason I had it in there was to check that my expect statement was accurately checking the data type of the argument.